default/scripting/techs/construction/FORCE_ENERGY_STRC.focs.py

```diff
@@ -16,68 +16,68 @@
             accountinglabel="CON_TECH_ACCOUNTING_LABEL",
             priority=AFTER_ALL_TARGET_MAX_METERS_PRIORITY,
             effects=[
-                If(
+                EffectIf(
                     condition=(Value(LocalCandidate.Industry) <= Value(LocalCandidate.TargetIndustry)),
                     effects=SetIndustry(
```

Comment:
perhaps make `effects=` be 'matching_effects` and `else=` be `non_matching_effects` ?

Reply:
I'd like to change original FOCS tokens and structure as little as possible.